splunk/src/main/java/com/splunk/HttpService.java

```diff
@@ -239,6 +234,14 @@ public static void setSslSecurityProtocol(SSLSecurityProtocol securityProtocol)
         }
     }
 
+    /**
+     * Adds list of Cluster Master Hosts to the list of Valid Hosts for Hostname verification.
+     * @param searchHeadService Splunk SearchHead Service instance
+     */
+    public static void setClusterMasterUri(Service searchHeadService){

```

Comment:
nit: Something more closely matching you comment like `addClusterMasterURIsToHosts()` might be a more appropriate name here

Reply:
yes agreed, I have updated the name. Thanks